### DISCUSSION TITLE
When will the new agreement come online

### DISCUSSION THREAD

**Discussion by Rempage** [Chia-Network/chia-blockchain#6877] [Support]
When will the new agreement come online

**Comment by BrandtH22** [comment]
@Rempage , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
